docs/api/changelog.rst

```diff
@@ -14,6 +14,7 @@ Changed
 ~~~~~~~
 - :class:`imod.mf6.Well` now also validates that well filter top is above well filter bottom
 - :function:`open_projectfile_data` now also imports well filter top and bottom.
+- :class:`imod.mf6.Well` now also validates that well filter top is above well filter bottom
```

Comment:
I see you planned to update the changelog, copy pasted a previous line, but forgot to update the message?